snippets/icon-account.liquid

```diff
@@ -1,4 +1,4 @@
-<svg
+{% comment %} <svg
```

Comment:
Commenting out the old icon is unnecessary and makes the git merge more messy. In this case, please just replace the old SVG with the custom SVG